tempodb/backend/block_meta.go

```diff
@@ -84,9 +84,8 @@ func (s DedicatedColumnScope) ToTempopb() (tempopb.DedicatedColumn_Scope, error)
 }
 
 type CompactedBlockMeta struct {
-	BlockMeta
-
 	CompactedTime time.Time `json:"compactedTime"`
+	BlockMeta
```

Comment:
accidental change? meaningful?

Reply:
Small but meaningful for the struct field alignment.